examples/detailed-tidb/tidb-cluster.yaml

```diff
@@ -0,0 +1,257 @@
+    #       - labelSelector:
+    #           matchExpressions:
+    #             - key: app.kubernetes.io/component
+    #               operator: In
+    #               values:
+    #                 - tidb
+    #         topologyKey: ""
+
+    ## Specify node labels for pods assignment
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    # nodeSelector:
+    #   app.kubernetes.io/component: tidb
+
+  tikv:
+    # Base image of the component
+    baseImage: pingcap/tikv
+
+    # TiKV Configuration
+    # https://pingcap.com/docs/tidb-in-kubernetes/stable/configure-cluster-using-tidbcluster/#configure-tikv-parameters
+    config: {}
+
+    # The desired ready replicas
+    replicas: 3
+
+    # describes the compute resource requirements.
+    # Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+    requests:
+      cpu: "100m"
+      storage: 1Gi
+
+    ## The storageClassName of the persistent volume for TiKV data storage.
+    ## Defaults to Kubernetes default storage class.
+    # storageClassName: ""
+
+    ## Affinity for pod assignment
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    # affinity:
+    #   podAntiAffinity:
+    #     preferredDuringSchedulingIgnoredDuringExecution:
+    #       - podAffinityTerm:
+    #           labelSelector:
+    #             matchExpressions:
+    #               - key: app.kubernetes.io/component
+    #                 operator: In
+    #                 values:
+    #                   - tidb
+    #                   - pd
+    #           topologyKey: ""
+    #         weight: 100
+    ## If you use require affinity, you must ensure that at least 3 nodes are available in the cluster
+    #     requiredDuringSchedulingIgnoredDuringExecution:
+    #       - labelSelector:
+    #           matchExpressions:
+    #             - key: app.kubernetes.io/component
+    #               operator: In
+    #               values:
+    #                 - tikv
+    #         topologyKey: ""
+  
+    ## Specify node labels for pods assignment
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    # nodeSelector:
+    #   app.kubernetes.io/component: tikv
+
+
+  ## ** TiDB advanced feature **
+  ## Deploy TiDB Binlog of a TiDB cluster
+  ## Ref: https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-tidb-binlog/#deploy-pump
+  # pump:
+  #   baseImage: pingcap/tidb-binlog
+  #   replicas: 1
+  #   storageClassName: local-storage
+  #   requests:
+  #     storage: 30Gi
+  #   schedulerName: default-scheduler
+  #   config:
+  #     addr: 0.0.0.0:8250
+  #     gc: 7
+  #     heartbeat-interval: 2
+
+  ## Ref: TiCDC is a tool for replicating the incremental data of TiDB
+  ## Ref: https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-ticdc/
+  # ticdc:
+  #   baseImage: pingcap/ticdc
+  #   replicas: 3
```

Comment:
I think we don't need to expose `cdc` and `binlog` in a detailed tidbcluster, there should be treated as a independent case.

Reply:
actually these features are not used, now compared to basic tidb cluster just add resource.request.